src/set_completed.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Collabora Limited
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
+
+import logging
+import os
+import sys
+
+import kernelci
+import kernelci.config
+import kernelci.db
+from kernelci.cli import Args, Command, parse_opts
+
+from logger import Logger
+
+
+class SetCompleted:
+
+    def __init__(self, configs, args):
+        self._db_config = configs['db_configs'][args.db_config]
+        api_token = os.getenv('API_TOKEN')
+        self._db = kernelci.db.get_db(self._db_config, api_token)
+        self._logger = Logger("config/logger.conf", "set_completed")
+
+    def run(self):
+        sub_id = self._db.subscribe_node_channel(filters={
+            'op': ('created', 'updated'),
+            'status': ('completed', 'timeout'),
+        })
+        self._logger.log_message(logging.INFO, "Listening for node events... ")
+        self._logger.log_message(logging.INFO, "Press Ctrl-C to stop.")
+        sys.stdout.flush()
```

Comment:
This can be dropped as the logger takes care of it.

Reply:
The documentation states that the `StreamHandler.flush()` should be called explicitly at times. But I failed to get any other precise information about this.
https://docs.python.org/3/library/logging.handlers.html#logging.StreamHandler.flush